stable/jenkins/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: jenkins
 home: https://jenkins.io/
-version: 0.16.1
+version: 0.16.2
```

Comment:
Technically, I believe this should be `0.17.0`, because you're adding a new feature, as opposed to fixing a bug.